src/pages/MonthlyGood/PersonalizedMonthlyGoodLandingPage.vue

```diff
@@ -231,7 +231,7 @@ export default {
 			this.amount = value;
 		},
 		submit() {
-			this.$kvTrackEvent('Monthly Good', 'click-personalized-MG-signup-cta', 'Get started');
+			this.$kvTrackEvent('MonthlyGood', 'click-personalized-MG-signup-cta', 'Get started');
```

Comment:
@BoulderBrains sorry about the flip flop on this. I've gone back in and removed the space here and on all my new events.